.circleci/config.yml

```diff
@@ -522,7 +522,7 @@ jobs:
     <<: *environment
     steps:
       - checkout
-      - *restore_cache
+      # - *restore_cache
```

Comment:
you shouldn't need to do this when you use the git has in requirements as you did.